src/devices/datek.ts

```diff
@@ -41,7 +41,22 @@ const definitions: DefinitionWithExtend[] = [
         vendor: 'Datek',
         description: 'Datek Eva AMS HAN power-meter sensor',
         fromZigbee: [fz.hw_version],
-        extend: [onOff(), electricityMeter({threePhase: true, fzMetering: fz.metering_datek, producedEnergy: true}), temperature()],
+        extend: [
+            electricityMeter({
+                cluster: 'metering',
+                fzMetering: fz.metering_datek,
+                producedEnergy: true,
+            }),
+            electricityMeter({
+                threePhase: true,
+                cluster: 'electrical',
+                power: false,
+                voltage: {},
+                current: {},
+                fzMetering: fz.metering_datek,
```

Comment:
I think this and voltage + current can be removed

Reply:
Agreee that the custom fromZigbee should only be set on the SE variant.

When it comes to voltage/current: These were previously (pre refactor) set up to report changes down to 0, while the default when extending is 5V and 0.05A respectively. Should I perhaps change them to `{change: 0}` instead of removing? I noticed that especially voltage becomes almost pointless when set to 5V, since the normal ripple around 230V is much smaller than that.

Reply:
>  I noticed that especially voltage becomes almost pointless when set to 5V, since the normal voltage variation around 230V is much smaller than that.

Maybe we should reduce the default value of 5V? What about 3?

Reply:
I guess it depends on what the intent of the default is: Is it to capture significant voltage changes (e.g. brownouts), or to keep real-time eye on a healthy power supply? 

This is what I currently have in Grafana, based on the full 0.1V precision of the meter:

<img width="1346" alt="Screenshot 2025-01-02 at 14 20 40" src="https://github.com/user-attachments/assets/5f20ef1e-141f-4b6a-b381-e1ea039bd838" />

Reply:
For now, I removed current and set voltage to `change: 0` to make the behavior for the Datek meter similar to how it worked in the past. Please let me know if you instead want me to change the default for all electricity meters.

Reply:
> Is it to capture significant voltage changes

We should find a healthy balance here, but change `0` use cases are very specific, if a users wants that it can always be changed manually via the reporting tab. I think all devices should use the same default (for consistency purposes)

Reply:
I'm completely fine with removing it if that matches the project goals. It just seemed like a regression to me for data that was previously streamed over MQTT to stop getting emitted after an update. I'll update this PR to leave it at the default, and let a discussion about potentially changing the default happen separately.